File: tugasKelompok/Fisika 2 (Energi Kinetik)/C++/energi_kinetik.cpp
#include <stdio.h>
#include <math.h>
int main(){
    int v, m;
    float ek;
    printf("\tMenghitung Energi Kinetik\n");
    printf("Masukan masa benda (Kg) "); scanf("%d", &m);
    printf("Masukan kecepatan benda (m/s) "); scanf("%d", &v);
    ek = float(0.5*v*pow(m, 2));
    printf("Energi kinetik dari benda dengan massa %d Kg dan kecepatan %d m/s adalah %.2f joule", m, v, ek);
    return 0;
}

File: Teori Daspro/tugasKelompok/Fisika 2 (Energi Kinetik)/Pascal/energi_kinetik.pas
program menghitung_energi_kinetik;
var 
    v, m: integer;
    ek: double;
begin
    writeln('Menghitung Energi Kinetik');
    write('Masukan massa benda (Kg) '); readln(m);
    write('Masukan kecepatan benda (m/s) '); readln(v);
    ek := 0.5*m*v*v;
    writeln('Energi kinetik benda dengan massa ',m,' dan kecepatan ',v,' adalah ',ek:5:2,' joule');
end.